src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -95,6 +96,23 @@ public static Email parseEmail(String email) throws ParseException {
         return new Email(trimmedEmail);
     }
 
+    /**
+     * Parses a {@code String deadline} into a {@code Deadline}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @param deadline the date of the deadline.
+     * @return the resulting {@code Deadline} object.
+     * @throws ParseException if the given {@code deadline} is invalid.
+     */
+    public static Deadline parseDeadline(String deadline) throws ParseException {
```

Comment:
I notice this method, as well as MESSAGE_INVALID_DATE_FORMAT is not used. Will you be using them in the future?

Reply:
Kind of forgot it was there, maybe will use in the future.